roles-in-tech/technical-writing.md

```diff
@@ -1,12 +1,8 @@
-# Example Topic Outline (Replace with Title)
+# Technical Writing
 
 ### Projected Time
 
-Example: 30-45 minutes
-- Lesson: ? min
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
+About 30 minutes
 
 ### Prerequisites
 
```

Comment:
Can you replace the example with our outline on writing good docs? 
https://github.com/Techtonica/curriculum/blob/master/reading-and-writing-documentation/documentation.md

Reply:
Sure, @alodahl, why not.
Done!

Reply:
Close... could you move that link to line 9 and remove the word "example"?

Reply:
Also, please change the typos pointed out yesterday.

Reply:
The requested changes are fixed . 😄